packages/Search/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -273,8 +240,9 @@ KOKKOS_INLINE_FUNCTION int queryDispatch(
     auto const geometry = pred._geometry;
     auto const k = pred._k;
     return nearestQuery( bvh,
-                         [geometry]( Node const *node ) {
-                             return distance( geometry, node->bounding_box );
+                         [geometry, bvh]( Node const *node ) {
```

Comment:
```C++
[geometry](typename BVH::bounding_volume const &other)
{ return distance( geometry, other ); }
```
and then you will need to change the calls `distance( node )` in the nearest traversal to `distance( bvh.getBoundingVolume(node) )`

Reply:
Modified accordingly.